src/vg_active_segment.erl

```diff
@@ -61,6 +82,27 @@ init([Topic, Partition, NextBrick, Tab]) ->
     {ok, Position} = file:position(LogFD, eof),
     {ok, IndexPosition} = file:position(IndexFD, eof),
 
+    lager:info("starting seg mgr next ~p", [NextServer]),
+
+    %% Trigger server start on next server
+    NextBrick =
+        case NextServer of
+            solo -> solo;
+            tail -> tail;
+            last -> last;
+            S when is_atom(S) ->
+                {ok, SupPid} = vonnegut_sup:create_topic(S, Topic, [Partition]),
```

Comment:
I'm not super sure about this.  It's probably better to have some administrative gen_server that manages these sorts of cluster-wide actions and manages the segment chain creation, rather than having it be implicit when the head is created.  Also I think that this is going to break if/when there are more than one partitions for a topic.